homeassistant/components/sensibo/entity.py

```diff
@@ -125,8 +130,13 @@ def __init__(
         )
 
     @property
-    def sensor_data(self) -> MotionSensor | None:
+    def sensor_data(self) -> MotionSensor:
         """Return data for Motion Sensor."""
         if TYPE_CHECKING:
             assert self.device_data.motion_sensors
         return self.device_data.motion_sensors[self._sensor_id]
+
+    @property
+    def available(self) -> bool:
+        """Return True if entity is available."""
+        return bool(self.sensor_data.alive) and super().available
```

Comment:
Why the bool()?

Reply:
`alive` could be `None` from typing so therefore using a bool to evaluate `None` to `False`